### PR TITLE
修改paddlecloud以便于PFSSever取得token

### PR DESCRIPTION
**Pull request by gongweibao**

### CODE REVIEW

File: paddlecloud/paddlecloud/urls.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
     url(r"^api/v1/logs/", paddlejob.views.LogsView.as_view()),
     url(r"^api/v1/workers/", paddlejob.views.WorkersView.as_view()),
     url(r"^api/v1/quota/", paddlejob.views.QuotaView.as_view()),
-    url(r"^api/v1/file/", paddlejob.views.SimpleFileView.as_view()),
+    url(r"^api/v1/token2user/", paddlejob.views.GetUserView.as_view()),
```
Comment (Yancey1989) [inline paddlecloud/paddlecloud/urls.py]: Why we delete the simplefile API?

Comment (gongweibao) [inline paddlecloud/paddlecloud/urls.py]: merge的时候出了点问题。非常感谢。
Done。

File: paddlecloud/paddlejob/views.py [hunk 2]
```diff
@@ -293,6 +293,16 @@ def get(self, request, format=None):
             .list_namespaced_resource_quota(namespace)
         return Response(quota_list.to_dict())
 
+class GetUserView(APIView):
+    permission_classes = (permissions.IsAuthenticated,)
+
+    def get(self, request, format=None):
+        content = {
+            'user': unicode(request.user.username),  # `django.contrib.auth.User` instance.
```
Comment (typhoonzero) [inline paddlecloud/paddlejob/views.py]: We use user's email as username all the time, and emails are strictly checked. Also using django's model values will return unicode typed strings by default.

So don't convert the type.

File: paddlecloud/notebook/views.py [hunk 3]
```diff
@@ -44,6 +44,7 @@ def get(self, request, format=None):
         }
         return Response(content)
 
+
```
Comment (typhoonzero) [inline paddlecloud/notebook/views.py]: What does this blank line do?

File: paddlecloud/paddlejob/views.py (hunk 2 above)
Comment (gongweibao) [inline paddlecloud/paddlejob/views.py]: Done

File: paddlecloud/notebook/views.py (hunk 3 above)
Comment (gongweibao) [inline paddlecloud/notebook/views.py]: Done.

**Pull request closed without merging**